scripts/deprecated-msg.js

```diff
@@ -0,0 +1,2 @@
+var chalk = require('chalk');
```

Comment:
i don't think we do. people will have to `npm install` with this pull. it's overkill IMO